packages/workspace/src/workspace/state/static_files_sources/history_static_files_source.ts

```diff
@@ -71,6 +71,9 @@ export const buildHistoryStateStaticFilesSource = (dirStore: StateStaticFilesSto
       log.debug('clear ignored in history state static files source')
     },
 
-    flush: () => log.timeTrace(() => dirStore.flush(), 'Flushing history static state files source'),
+    flush: () =>
+      log.timeTrace(async () => {
+        await dirStore.flush()
+      }, 'Flushing history static state files source'),
```

Comment:
nit
```suggestion
      }, 'historyStaticFilesSource.flush'),
```
would make log parsing handle this a bit better